config/jobs/build-drivers/build-new-centos.yaml

```diff
@@ -147,35 +147,6 @@ postsubmits:
             memory: 6Gi
       nodeSelector:
         Archtype: "x86"
-  - name: build-new-drivers-centos-2-postsubmit-arm
```

Comment:
We only support kernels >= 3.16 on arm; avoid having a useless centos2.X job.

Reply:
https://github.com/falcosecurity/libs#drivers-officially-supported-architectures